cl/phase1/network/services/batch_signature_verification.go

```diff
@@ -40,6 +41,10 @@ type AggregateVerificationData struct {
 	GossipData *sentinel.GossipData
 }
 
+func init() {
+	batchCheckInterval.Store(500)
```

Comment:
????

Reply:
yeah no keep this as normal - i am definetely not doing this just for a race warn